komal/Accordion/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import Accordion from './components/Accordion'
+
+const items = [
+  {
+    title: 'What is React?',
+    content: 'React is Fornt end Javascript framework'
+  },
+  {
+    title: 'Why use React?',
+    content: 'React is a favorite JS library amony engineers'
+  },
+  {
+    title: 'How do we use React?',
+    content: 'We use React by creating components' 
+  }
+]
+
+export default  () => {
+  return(
+    <div>
+      <Accordion items={items} />
+    </div>
+  ) 
+}
```

Comment:
Please verify if this is throwing a warning in the js console.

Reply:
yes, it is giving warning on js console. i will fix this code.